packages/studio-web/project.json

```diff
@@ -49,8 +49,8 @@
           "budgets": [
             {
               "type": "initial",
-              "maximumWarning": "500kb",
-              "maximumError": "2mb"
+              "maximumWarning": "1500kb",
+              "maximumError": "3mb"
```

Comment:
We were busting our 2mb budget deploying this, I think it's reasonable to go to 3mb, that's still reasonably small.